ui/lib/core/addon/components/radio-select-ttl-or-string.hbs

```diff
@@ -34,14 +36,13 @@
     </p>
     {{#if (eq this.groupValue "specificDate")}}
       <input
-        data-test-input="not_after"
         id="not_after"
         autocomplete="off"
         spellcheck="false"
         value={{this.notAfter}}
         {{on "input" this.setAndBroadcastInput}}
         class="input"
-        maxLength="21"
```

Comment:
my original thought here was that a utc timestamp wouldn't go past this length, but that's not true. To stay safe, I just removed the restriction.